<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="gameState">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>511</width>
      <height>291</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QGroupBox" name="principalBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>491</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QTextEdit" name="pregunta">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>391</width>
         <height>51</height>
        </rect>
       </property>
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="respA">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>20</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="respB">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>20</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="respC">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>120</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>20</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="respD">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>180</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>20</number>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxA">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>120</y>
         <width>16</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxB">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>180</y>
         <width>16</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxC">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>120</y>
         <width>16</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxD">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>180</y>
         <width>16</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="labelRespB">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Respuesta B:</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelRespA">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Respuesta A:</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelRespC">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>100</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Respuesta C:</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelRespD">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>160</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Respuesta D:</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelPregunta">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Pregunta 1:</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelCantResp">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>50</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Cantidad de&lt;br/&gt;respuestas:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="cantRespuestas">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>90</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>4</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
      <widget class="QPushButton" name="pregButton_7">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>240</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pregButton_4">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>240</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pregButton_6">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>240</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pregButton_3">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>240</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pregButton_1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pregButton_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>240</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pregButton_5">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>240</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
      <widget class="QPushButton" name="startButton">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>215</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Empezar partida</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="buttonicons.qrc">
         <normaloff>:/icons/playButton.png</normaloff>:/icons/playButton.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="saveButton">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>215</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Guardar partida</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="buttonicons.qrc">
         <normaloff>:/icons/saveButton.png</normaloff>:/icons/saveButton.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="configButton">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>215</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Configuracion</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="buttonicons.qrc">
         <normaloff>:/icons/configButton.png</normaloff>:/icons/configButton.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="newPregButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>215</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Nueva pregunta</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="buttonicons.qrc">
         <normaloff>:/icons/plusButton.png</normaloff>:/icons/plusButton.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="removeButton">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>215</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Borrar pregunta</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="buttonicons.qrc">
         <normaloff>:/icons/minusButton.png</normaloff>:/icons/minusButton.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="previsualizarButton">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>215</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Previsualizar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="closeButton">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>0</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Cerrar juego</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="buttonicons.qrc">
         <normaloff>:/icons/closebutton.png</normaloff>:/icons/closebutton.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="mensajeError" native="true">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>-90</y>
        <width>221</width>
        <height>91</height>
       </rect>
      </property>
      <widget class="QLabel" name="msgErrorGuardar">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>181</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#ff0000;&quot;&gt;Error al guardar&lt;/span&gt;&lt;br/&gt;Debe escribir la pregunta,&lt;br/&gt;sus respuestas y seleccionar&lt;br/&gt;a correcta.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="msgErrorComunicacion">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>181</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#ff0000;&quot;&gt;Error de comunicacion&lt;/span&gt;&lt;br/&gt;Debe configurar el puerto&lt;br/&gt;antes de empezar la partida.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="mensajeGuardar" native="true">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>-40</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>6</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Pregunta guardada&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QGroupBox" name="gameInProgressBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>491</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>501</width>
         <height>171</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>40</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Juego en progreso</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>491</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="closeButton_2">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>240</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Cerrar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="replayButton">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>240</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Volver a jugar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>211</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Puntajes:</string>
       </property>
      </widget>
      <widget class="QLabel" name="puntajePrimerPuesto">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>351</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;p&gt;1.- Jugador # : &lt;strong&gt;# Puntos&lt;/strong&gt;&lt;/p&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="puntajeSegundoPuesto">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>351</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;p&gt;2.- Jugador # : &lt;strong&gt;# Puntos&lt;/strong&gt;&lt;/p&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="puntajeTercerPuesto">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>80</y>
         <width>351</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;p&gt;3.- Jugador # : &lt;strong&gt;# Puntos&lt;/strong&gt;&lt;/p&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="puntajeCuartoPuesto">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>110</y>
         <width>351</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;p&gt;4.- Jugador # : &lt;strong&gt;# Puntos&lt;/strong&gt;&lt;/p&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="puntajeQuintoPuesto">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>140</y>
         <width>351</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;p&gt;5.- Jugador # : &lt;strong&gt;# Puntos&lt;/strong&gt;&lt;/p&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="puntajeSextoPuesto">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>170</y>
         <width>351</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;6.- Jugador # : &lt;span style=&quot; font-weight:600;&quot;&gt;# Puntos&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="puntajeSeptimoPuesto">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>200</y>
         <width>351</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;7.- Jugador # : &lt;span style=&quot; font-weight:600;&quot;&gt;# Puntos&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="puntajeOctavoPuesto">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>230</y>
         <width>351</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;8.- Jugador # : &lt;span style=&quot; font-weight:600;&quot;&gt;# Puntos&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="buttonicons.qrc"/>
 </resources>
 <connections/>
</ui>
